CHANGELOG

```diff
@@ -31,14 +48,22 @@ Note this release was not merged to main before deploying.
 
 ## Added
 
+- Add server-side EDD form posting function (SCC-3760)
+- Add EDD request page and field validation logic (SCC-3760)
 - Holding location filters to refine search and advanced search panels
 - "Go back" button to advanced search
+- Add server side delivery location fetch and location utils (SCC-3759)
+- Add holding location filters to refine search and advanced search panels
+- Add go back button to advanced search
 
-## Updated
+### Updated
 
+- Better handling of bibs with no titles in SearchResultsBib model
+- Moved closedLocations from environment variables to appConfig (SCC-3759)
```

Comment:
I think only line 62 and 66 should go to prerelease. Line 61 was already in the changelog.